docker/scripts/setup

```diff
@@ -29,7 +29,7 @@ if [ -d "/scripts/cache" ]; then
   mv /scripts/cache vendor/
   chown -R huginn:huginn vendor/cache
 fi
-sudo -u huginn -H bundle install --deployment --without development test
+sudo -u huginn -H bundle install --deployment --with deployment
```

Comment:
Looks like it's not that easy I am still a bit confused with all the PORT changes, trying to figure it out :smile: